blueocean-github-pipeline/pom.xml

```diff
@@ -57,8 +57,8 @@
     </dependency>
 
     <dependency>
-      <groupId>com.github.tomakehurst</groupId>
-      <artifactId>wiremock-jre8-standalone</artifactId>
+      <groupId>org.wiremock</groupId>
```

Comment:
Not strictly necessary for this PR, but getting rid of remaining Java 8 references as we move to Java 17 felt wise.